### PR TITLE
Post for 2.7.0 release

### PR DESCRIPTION
**Pull request by mikaelrandy**

### CODE REVIEW

**Comment by mikaelrandy** [comment]
ping @Hywan @jubianchi

**Comment by mikaelrandy** [comment]
As atoum 2.7.0 is released from 2 days, i merge ;)

File: _posts/2016-05-20-release-2-7-0.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+author: "@atoum/RMs"
+categories: release
+---
+
+## Stats
+
+* 13 commits,
+* 82 files changed,
+* 4 contributors,
+* 1 new features,
```
Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:14]:
``` diff
- features
+ feature
```

File: _posts/2016-05-20-release-2-7-0.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* 2 bug fixes.
+
+## What's new
+
+### Use telemetry by atoum itself
+
+As we recently announced, we now provide a collaborative platform to share tests results and statistics called [Telemetry].
```
Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:21]: Please, add a link on “recently announced”.

Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:21]:
``` diff
- tests results
+ test results
```

File: _posts/2016-05-20-release-2-7-0.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+According to the atoum's philosophy, atoum itself tests execution now send results to the telemetry platform.
```
Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:23]:
``` diff
- tests execution
+ test executions
```

File: _posts/2016-05-20-release-2-7-0.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Bugfixes
+
+### Reports override correctly when using -ulr/-utr
```
Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:27]:
``` diff
- -ulr/-utr
+ `-ulr`/`-utr`
```

File: _posts/2016-05-20-release-2-7-0.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Using `--use-light-report` or `--use-tap-report` no longer remove all defined reports (like Telemetry) but only those who are in conflict with light or TAP reports.
+
+### Assertions on PHP 7 exceptions/throwables/errors are now working correctly
```
Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:31]: I would have written: “Assertions are now working expectedly on PHP7”. No need to list them in the title.

Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:31]: OK, so it's just about `Throwable` & co. So new proposal: “Exception asserter handles new PHP7 features”. Better I guess.

File: _posts/2016-05-20-release-2-7-0.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+`->exception()` assert handle correctly PHP7 `\Exception`, `\Throwable` and `\TypeErrors` exceptions
```
Comment (Hywan) [inline _posts/2016-05-20-release-2-7-0.md:33]:
````````` diff
- `->exception()`
+ `exception`
``````diff
- `\Exception`
+ `Exception`
`````````

``` diff
- `\Throwable`
+ `Throwable`
```

``` diff
- `\TypeErrors`
+ `TypeError`
```

In addition to links to the documentation.

**Comment by Hywan** [comment]
Damn, didn't see it was merged. Can you still apply my comment?

**Comment by mikaelrandy** [comment]
@Hywan fixed here : https://github.com/atoum/atoum.github.io/commit/add7c1ec4624917cc7b8fd19972499d3092c6040

**Comment by Hywan** [comment]
:+1:

**Pull request merged**